packages/features/ee/dsync/page/user-dsync-view.tsx

```diff
@@ -0,0 +1,32 @@
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
+import { HOSTED_CAL_FEATURES } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Meta } from "@calcom/ui";
+
+import { getLayout } from "../../../settings/layouts/SettingsLayout";
+import ConfigureDirectorySync from "../components/ConfigureDirectorySync";
+
+// For Self-hosted Cal
+const DirectorySync = () => {
```

Comment:
how can I get to that?

Reply:
There's no logic for to provision users for self-hosters. We wanted to focus on getting this shipped for orgs right now.

Reply:
should we remove that for now then? It's just showing an error for me when `HOSTED_CAL_FEATURES` is not set:
![Screenshot 2024-03-04 at 3 30 09 PM](https://github.com/calcom/cal.com/assets/30310907/75f26e3e-840b-4864-97e4-f45f4e9da5b6)